### PR TITLE
Fix mobile header style error

### PR DESCRIPTION
**Pull request by academey**

### CODE REVIEW

File: app/components/home/actions.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import axios from "axios";
 import * as moment from "moment";
 import { ACTION_TYPES } from "../../actions/actionTypes";
 import { Dispatch } from "react-redux";
+// import FileNameMaker from "../../helpers/fileNameMaker";
```
Comment (TylerShin) [inline app/components/home/actions.tsx]: If you don't use this helper, just remove this line.

File: app/components/home/index.tsx [hunk 2]
```diff
@@ -160,6 +163,54 @@ class HomeComponent extends React.PureComponent<IHomeComponentProps, {}> {
...
+      fileName = await dispatch(
+        Actions.uploadImage({
+          imageDataURL: imageUrl
+        })
+      );
+    } catch (err) {
+      console.error(err);
+    }
+
+    const plutoUrlWithImage = encodeURIComponent(`https://join.pluto.network/userImage/${fileName}`);
```
Comment (TylerShin) [inline app/components/home/index.tsx:181]: :+1:

File: app/components/home/index.tsx [hunk 3]
```diff
@@ -160,6 +163,54 @@ class HomeComponent extends React.PureComponent<IHomeComponentProps, {}> {
...
+        const background = new Image();
+        const backgroundUrl =
+          "https://images.pexels.com/photos/104827/cat-pet-animal-domestic-104827.jpeg?h=350&auto=compress&cs=tinysrgb";
+        this.drawingCanvas.width = 600;
+        this.drawingCanvas.height = 314;
+        background.setAttribute("crossOrigin", "anonymous");
+        background.src = backgroundUrl;
+        background.onload = () => {
+          context.drawImage(background, 0, 0);
+          context.font = "20px Roboto";
```
Comment (TylerShin) [inline app/components/home/index.tsx:202]: I guess that if the user's browser doesn't have Roboto font, the font won't be displayed as we expected

**Review by TylerShin — CHANGES_REQUESTED** [review]

File: app/components/home/index.tsx (hunk 3 above)
Comment (academey) [reply app/components/home/index.tsx:202]: That site use roboto font. So maybe it can work after downloading font

**Pull request merged**